txscript/standard_test.go

```diff
@@ -1013,3 +1013,28 @@ func TestStringifyClass(t *testing.T) {
 		}
 	}
 }
+
+// TestNullDataScript tests whether NullDataScript returns a valid script.
+func TestNullDataScript(t *testing.T) {
+	data := []byte{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
```

Comment:
Nice.  We just found a bug in the `GetScriptClass` function.  I confirmed that 6a51 is indeed showing up as `nonstandard`, but it most certainly is a `nulldata` script.

```
{
  "asm": "OP_RETURN 1",
  "type": "nulldata",
  "p2sh": "35zfU45S1vbHDhfY3ibN8XgCafBUzTvT3a"
}
```